wagtail/admin/tables.py

```diff
@@ -0,0 +1,190 @@
+"""Helper classes for formatting data as tables"""
```

Comment:
Why not put this in ``wagtail/admin/ui/``?

Reply:
Sounds good! Wasn't sure if admin/ui was intended for more front-end / react-y things, but if you think it's a good fit, let's go with that.

Reply:
Sounds good! Wasn't sure if admin/ui was intended for more front-end / react-y things, but if you think it's a good fit, let's go with that.